crates/core/build.rs

```diff
@@ -8,6 +8,7 @@ use std::{
 use inflector::Inflector;
 
 fn main() {
+    uniffi::generate_scaffolding("src/lib.udl").unwrap();
```

Comment:
I guess you just ignored my input on this one. There are macros for this now; you're doing this in an outdated and harder to maintain way